src/util/domain-icon.js

```diff
@@ -38,6 +38,9 @@ export default function domainIcon(domain, state) {
     case 'input_select':
       return 'mdi:format-list-bulleted';
 
+    case 'input_slider':
+      return 'mdi:volume-high';
```

Comment:
This is not a good domain icon for input_slider. People expect this icon to mean volume. What about `mdi:ray-vertex` or `mdi:sort-numeric` ?